user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.js

```diff
@@ -1180,29 +1257,17 @@ define([
         var path = $target.attr("path");
         var myself = this;
 
-        var url = CONTEXT_PATH + "api/repo/files/" +
-            FileBrowser.encodePathComponents(path == null ? ":" : Encoder.encodeRepositoryPath(path))
-            + "/tree?depth=1&showHidden=" + myself.model.get("showHiddenFiles") + "&filter=*%7CFOLDERS";
+        var url = CONTEXT_PATH + "plugin/scheduler-plugin/api/generic-files/" +
+            FileBrowser.encodePathComponents(path == null ? ":" : encodeGenericPath(path))
```

Comment:
I know this was here already, but could you change this to the following, to keep the encoding logic within the encoding function:
```js
FileBrowser.encodePathComponents(encodeGenericPath(path == null ? "/" : path))
```

Reply:
In a previous comment, I had put into question that we'd need to check for null in the first place: https://github.com/pentaho/pentaho-platform/pull/5562#discussion_r1534477184.

Reply:
made first comment change, looking into commit history to see if I can understand how/why we could be expanding a folder of null path 🤔

Reply:
It was added in the same [commit](https://github.com/pentaho/pentaho-platform/commit/499f11cd98be2c642cea34c3ee4eeb8f6686304d) that introduced the HTML block that ignored the handlebars framework (that I fixed below). I feel confident removing it.